jwst/pipeline/calwebb_spec3.py

```diff
@@ -2,7 +2,7 @@
 
 from ..stpipe import Pipeline
 from .. import datamodels
-from ..exp_to_source import exp_to_source
+from ..exp_to_source import multislit_to_container
 
 # step imports
 from ..skymatch import skymatch_step

```

Comment:
Mihai's MRS background matching step is now in place, but bears a different name. You'll need:

from ..mrs_imatch import mrs_imatch_step

with similar updates to the list of step defs.